pkg/annotation/aws.go

```diff
@@ -1,5 +1,15 @@
 package annotation
 
+// AWSCNIMinimumIPTarget is the annotation to configure value for MINIMUM_IP_TARGET for AWS CNI
+// see https://github.com/aws/amazon-vpc-cni-k8s#cni-configuration-variables
+// and https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/eni-and-ip-target.md
+const AWSCNIMinimumIPTarget = "alpha.aws.giantswarm.io/cni-minimum-ip-target"
+
+// AWSCNIWarmIPTarget is the annotation to configure value for WARM_IP_TARGET for AWS CNI
+// see https://github.com/aws/amazon-vpc-cni-k8s#cni-configuration-variables
+// and https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/eni-and-ip-target.md
+const AWSCNIWarmIPTarget = "alpha.aws.giantswarm.io/cni-warm-ip-target"
```

Comment:
What about this structure? I feel like this is a more appropriate structure for the domain namespace. 

```suggestion
const AWSCNIWarmIPTarget = "cni.alpha.aws.giantswarm.io/warm-ip-target"
```

Reply:
@giantswarm/sig-ux @marians  WDYT?

Reply:
I don't think we have system regarding annotation grouping, and I can't judge if we need this group.

Just noting, the prefix could be either of

- `cni.alpha.aws.giantswarm.io/`
- `alpha.cni.aws.giantswarm.io/`

Reply:
`alpha.cni.aws.giantswarm.io/` would make more sense for me

Reply:
but uncertain since we don't use it anywhere else so far

Reply:
Then I vote for `alpha.cni.aws.giantswarm.io/`, too